### PR TITLE
Migrate NUM_NODES to --gcp-nodes for scalability jobs

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -3206,6 +3206,7 @@
       "--extract=ci/latest",
       "--gcp-master-image=gci",
       "--gcp-node-image=gci",
+      "--gcp-nodes=2000",
```
Comment (shyamjvs) [inline jobs/config.json]: 2001? (with the heapster node)

**Comment by shyamjvs** [comment]
Also to fix the presubmit, you probably want to run `bazel run //jobs:config_sort`

File: jobs/config.json (hunk 1 above)
Comment (wojtek-t) [reply jobs/config.json]: I changed that on purpose (for 5000-node we also have 5000, not 5001).

Comment (shyamjvs) [reply jobs/config.json]: Ok, SG

**Comment by wojtek-t** [comment]
Should be fixed now - PTAL

**Comment by shyamjvs** [comment]
/lgtm

Thanks!

File: jobs/env/ci-kubernetes-e2e-gce-large-manual-up.env [hunk 2]
```diff
@@ -2,7 +2,6 @@
 CLOUDSDK_CORE_PRINT_UNHANDLED_TRACEBACKS=1
 
 ### cluster-env
-NUM_NODES=1000
```
Comment (shyamjvs) [inline jobs/env/ci-kubernetes-e2e-gce-large-manual-up.env]: Seems like you forgot to add this to the job's `--gcp-node`.

Comment (shyamjvs) [reply jobs/env/ci-kubernetes-e2e-gce-large-manual-up.env]: Sent https://github.com/kubernetes/test-infra/pull/7712 to fix.

**Pull request merged**